scripts/infinity_grid.py

```diff
@@ -101,7 +101,29 @@ def apply_prompt_replace(p, v):
     if Script.VALIDATE_REPLACE and match not in p.prompt and match not in p.negative_prompt:
         raise RuntimeError(f"Invalid prompt replace, '{match}' is not in prompt '{p.prompt}' nor negative prompt '{p.negative_prompt}'")
     p.prompt = p.prompt.replace(match, replace)
-    p.negative_prompt = p.negative_prompt.replace(match, replace)
+
+#I didnt realize that the prompt replace function(s) had changed. my version still uses an older method of doing it. I will just copy yours and update to be negative for now. 
+def negative_prompt_replace_parse_list(in_list):
```

Comment:
this method is just a duplicate of the existing method but with a single string changed? could just be a parameter in the method